release/models/policy-forwarding/openconfig-pf-srte.yang

```diff
@@ -53,6 +59,30 @@ module openconfig-pf-srte {
   oc-ext:catalog-organization "openconfig";
   oc-ext:origin "openconfig";
 
+
+  typedef explicit-null-label-type {
+    type enumeration {
+      enum IPV4 {
+        description "Push an IPv4 Explicit NULL label on an unlabeled 
```

Comment:
The layout for descriptions through OpenConfig models is:

```yang
enum IPV4 {
  description
     "Blah blah blah";
}
```
Please reformat here :-)

Reply:
Done